test/standalone/issue_7030/main.zig

```diff
@@ -1,5 +1,17 @@
 const std = @import("std");
 
+pub fn log(
+    comptime message_level: std.log.Level,
+    comptime scope: @Type(.EnumLiteral),
+    comptime format: []const u8,
+    args: anytype,
+) void {
+    _ = message_level;
+    _ = scope;
+    _ = format;
+    _ = args;
+}
+
 pub fn main() anyerror!void {
     std.log.info("All your codebase are belong to us.", .{});
```

Comment:
This is a very good example too. This test was basically made to do nothing, which is confusing. Now with this empty `log` above it might be clearer that this doesn't actually log anything.